### ISSUE TITLE
FUNCTION_INVOCATION_TIMEOUT when using Prisma Data Platform

### ISSUE THREAD

**Issue by nacho-carnicero** [prisma/prisma#10429]
### Bug description

I am trying to use Prisma Data Platform to manage the connection to a DB on an app using serverless functions to execute backend actions. I have properly configured my app to connect to the connection string generated for my project in the Prisma Cloud. Normal actions seem to be working, but when I try to execute many actions in a short period of time my queries start to fail with the following message:

```
An error occurred with your deployment

FUNCTION_INVOCATION_TIMEOUT
```

<img width="1680" alt="Screenshot 2021-11-24 at 18 27 02" src="https://user-images.githubusercontent.com/17384901/143299244-fa65a556-0f83-44af-b691-1f780b4018ab.png">

### How to reproduce

1. Configure an app to connect to an endpoint of Prisma Data Platform
2. Send many queries to the Data Platform endpoint

### Expected behavior

The queries should not fail.

### Environment & setup

- OS: Mac OS
- Database: PostgreSQL
- Node.js version: v14.15.4

### Prisma Version

```
3.5.0-integration-fix-bundlers-broken-require.2
```

**Comment by pantharshit00** [comment]
Hello. 

Just a general question before we can look deeper here. How far is your database from the proxy. The region of the proxy can be seen in the prisma data platform dashboard. Please make sure database as close to it as possible .

**Comment by cebuladev** [comment]
`FUNCTION_INVOCATION_TIMEOUT` doesn't come from Prisma Client for sure - where do you deploy your functions? (which provider?)

Please enable logging (https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/logging) and share what happens under the hood.

**Comment by janpio** [comment]
Ping @nacho-carnicero - is this still a problem? Otherwise please close the issue. Thanks.

**Comment by janpio** [comment]
Closing, as no reaction. If anyone has a similar problem, please open a new issue. Thanks!
